test/runtests.jl

```diff
@@ -4,50 +4,44 @@ using SafeTestsets
-    @time @safetestset "ODE System Solving" begin include("solve_ODEs.jl") end
-    @time @safetestset "Make Jacobian" begin include("make_jacobian.jl") end
-    #@time @safetestset "DiffEq Steady State Solving" begin include("steady_state_problems.jl") end
-
-    # Tests related to solving Stochastic Differential Equations.
-    @time @safetestset "SDE System Solving" begin include("solve_SDEs.jl") end
-
-    # Tests related to solving Jump Systems.
-    @time @safetestset "Jump System Solving" begin include("solve_jumps.jl") end
-
-    # Tests related to PDESystems
-    @time @safetestset "PDE Systems" begin include("pdes.jl") end
-
-    # Miscellaneous tests
-    #@time @safetestset "Basic Plotting" begin include("plotting.jl") end
-    @time @safetestset "Latexify" begin include("latexify.jl") end
-
+    ### Tests the properties of ReactionSystems. ###
+    @time @safetestset "ReactionSystem Structure" begin include("reactionsystem_structure/reactionsystem.jl") end
+    @time @safetestset "Higher Order Reactions" begin include("reactionsystem_structure/higher_order_reactions.jl") end
+
+    ### Tests model creation via the @reaction_network DSL. ###
+    @time @safetestset "Basic DSL" begin include("dsl/dsl_basics.jl") end
+    @time @safetestset "DSL Model Construction" begin include("dsl/dsl_model_construction.jl") end
+    @time @safetestset "Custom CRN Functions" begin include("dsl/custom_functions.jl") end
+    @time @safetestset "DSL Options" begin include("dsl/dsl_options.jl") end
+
+    ### Non-DSL model cration and modication. ###
+    @time @safetestset "ReactionSystem Components Based Creation" begin include("programmatic_model_creation/component_based_model_creation.jl") end
+    @time @safetestset "Programmatic Model Expansion" begin include("programmatic_model_creation/programmatic_model_expansion.jl") end
+
+    # Runs various miscellaneous tests.
+    @time @safetestset "API" begin include("miscellaneous_tests/api.jl") end
+    @time @safetestset "Symbolic Stoichiometry" begin include("miscellaneous_tests/symbolic_stoichiometry.jl") end
+    @time @safetestset "Events" begin include("miscellaneous_tests/events.jl") end
+    @time @safetestset "Units" begin include("miscellaneous_tests/units.jl") end
+
+    ### Tests ODE, SDE, PDE, and Gillespie Simulations. ###
+    @time @safetestset "ODE System Simulations" begin include("model_simulation/simulate_ODEs.jl") end
+    @time @safetestset "Automatic Jacobian Construction" begin include("model_simulation/make_jacobian.jl") end
+    @time @safetestset "U0 and Parameters Input Variants" begin include("model_simulation/u0_n_parameter_inputs.jl") end
+    @time @safetestset "DiffEq Steady State Solving" begin include("model_simulation/solve_steady_state_problems.jl") end
+    @time @safetestset "SDE System Simulations" begin include("model_simulation/simulate_SDEs.jl") end
+    @time @safetestset "PDE Systems Simulations" begin include("model_simulation/simulate_PDEs.jl") end
```

Comment:
I'd order these ODEs, SDEs, jumps, steady-state, PDEs. The last two are the least used, and so should be later in the testing (just to make it a bit less of a wait to see if an ODE/SDE/jump test fails).

Reply:
sounds good, will change.